### ISSUE TITLE
Loss not normalized by batch size

### ISSUE THREAD

**Issue by khanhptnk** [pytorch/examples#106]
After switching to batch first, .size(1) is no longer the batch size. 

https://github.com/pytorch/examples/blob/master/OpenNMT/train.py#L123

**Comment by msaroufim** [comment]
Link now 404 so this looks like a stale issue
